### ISSUE TITLE
[Enhancement Request] Enable Addon/Widgets Item Limit - lower than 20 for TMDB Lists

### ISSUE THREAD

**Issue by TwilightMercy** [Tikipeter/repository.tikipeter#254]
Hi,

This is my current setting in Navigation tab:

![image](https://github.com/Tikipeter/repository.tikipeter/assets/29895680/76aeed20-9eac-46a1-a7a0-eab192546554)

I see that the limits only applies to non-TMDB lists (such as "Trending" for example). Is it possible to control their lists and get below 20 for each page?

Thank you for your work

**Comment by ivarbrandt** [comment]
I don't think that is possible because it pulls a default 20 items per page from the TMDB API.

**Comment by TwilightMercy** [comment]
Maybe separate the items locally in FEN? if possible
